src/event.rs

```diff
@@ -1146,6 +1154,22 @@ impl PartialEq for InnerSizeWriter {
     }
 }
 
+/// The request to change the inner size synchronously was ignored.
+///
+/// See [`InnerSizeWriter::request_inner_size`] for details.
+#[derive(Debug, Clone)] // Explicitly not other traits, in case we want to extend it in the future
+pub struct RequestIgnored {
```

Comment:
@kchibisov asked in https://github.com/rust-windowing/winit/pull/3067#discussion_r1318159213:

> What's the reason this error is here and not on the `window` for example?

Because it's specific to `InnerSizeWriter`, which is also in this module.